docs/PATCHEXPLAIN.md

```diff
@@ -238,6 +267,44 @@ Below is an explanation of what Kexts OpenCore Legacy Patcher will inject into m
 * SMC-Spoof
   * Reason: Spoofs SMC version to 9.9999
   * Models: All models require when spoofing minimal or higher
+* AppleRAIDCard
+  * Adds AppleRaidCard Support for Xserves
+* AMDGPUWakeHandler
+  * Reason: Adds Software Based Demux for 2011 15/17 Macbook Pros
+* AppleIntelCPUPowerManagement
+  * Reason: Restores Ivy Bridge and older CPU Power Management
+* AppleIntelCPUPowerManagementClient
```

Comment:
Might be easier to just say `AppleIntelCPUPowerManagement and AppleIntelCPUPowerManagementClient`, instead of repeating twice